content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -41,11 +57,23 @@ def self.get_runners_count_by_ems(ems, factory_config)
               transformation_hosts(ems, factory_config).inject(0) { |sum, thost| sum + thost[:runners][:current] }
             end
 
+            def self.ems_max_runners(ems, factory_config, handle = $evm)
+              if ems.custom_get('Max Transformation Runners')
+                handle.log(:info, "Using max transformation runners from EMS custom attribute: #{ems.custom_get('Max Transformation Runners')}")
+                ems.custom_get('Max Transformation Runners').to_i
+              elsif factory_config['ems_max_runners']
+                handle.log(:info, "Using max transformation runners from factory config: #{factory_config['ems_max_runners']}")
+                factory_config['ems_max_runners'].to_i
+              else
+                handle.log(:info, "Using default max transformation runners: #{DEFAULT_EMS_MAX_RUNNERS}")
+                DEFAULT_EMS_MAX_RUNNERS
```

Comment:
The default value was 1 in the `get_transformation_host` previously here https://github.com/ManageIQ/manageiq-content/pull/379/files#diff-44115b1bf3b0e7b502024d527948ef6dL46

now `DEFAULT_EMS_MAX_RUNNERS = 10` which is what `VMTransform` expects.

Does the value need to be flexible?

If so I would suggest adding another option to the end of the `ems_max_runners` method.

Example: `def self.ems_max_runners(ems, factory_config, handle = $evm, max_runners = DEFAULT_EMS_MAX_RUNNERS)`

and override the value from `get_transformation_host`.

Reply:
Good idea. Indeed, it may change over time. And we may see a more complex heuristic to determine max_runners. I changed it also for host_max_runners.